Beiwe/Recline/Recline.swift

```diff
@@ -15,10 +15,8 @@ let kReclineMetadataKey = "reclineMetadata"
 /// A database ~manager class for CouchbaseLite
 class Recline {
     static let shared = Recline()  // singleton instance
-    
-    var manager: CBLManager!  // Top-level CouchbaseLite object
-    var db: CBLDatabase?      // CouchbaseLite database
-    var typesView: CBLView?   // persistent index, view can be queried using a CBLQuery.
+
+    var db: Database?
```

Comment:
Cool, all for simplifying this, but I don't know what that stuff was to start with....

Reply:
I'm not sure either, I think they are the old way of initializing Couchbase database.